app/src/main/java/com/infomaniak/drive/ui/fileList/preview/PDFDocumentAdapter.kt

```diff
@@ -0,0 +1,84 @@
+/*
+ * Infomaniak kDrive - Android
+ * Copyright (C) 2024 Infomaniak Network SA
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package com.infomaniak.drive.ui.fileList.preview
+
+import android.os.Bundle
+import android.os.CancellationSignal
+import android.os.ParcelFileDescriptor
+import android.print.PageRange
+import android.print.PrintAttributes
+import android.print.PrintDocumentAdapter
+import android.print.PrintDocumentInfo
+import java.io.FileInputStream
+import java.io.FileOutputStream
+import java.io.InputStream
+import java.io.OutputStream
+
+class PDFDocumentAdapter(
+    private val fileName: String,
+    private val file: java.io.File
```

Comment:
```suggestion
    private val file: java.io.File,
```

Reply:
Also, we use a TypeAlias in kDrive, to differentiate our File from Java File.

`typealias IOFile = java.io.File` _(in `Extensions.kt`)_

Reply:
I'm gonna do a check of the whole project.

Reply:
Check is done: https://github.com/Infomaniak/android-kDrive/pull/1254